main.py

```diff
@@ -949,6 +949,10 @@ def patch_record_in_place(fn, record, subdir):
         if dep.startswith("zstd >=1.4."):
             depends[i] = dep.split(",")[0] + ",<1.5.0a0"
 
+        # curl >=8.0.0 is not actually a major upgrade
+        if dep.startswith("libcurl >= 7."):
```

Comment:
```suggestion
        if dep.startswith("libcurl >=7.") or dep.startswith("curl >=7."):
```
Removed the space after =. Also, a few packages have this constraint on curl itself.
https://metayaml-conda.fly.dev/metayaml/reverse_dependencies?name=curl